activestorage/app/javascript/activestorage/ujs.js

```diff
@@ -42,29 +41,12 @@ function handleFormSubmissionEvent(event) {
       if (error) {
         inputs.forEach(enable)
       } else {
-        submitForm(form)
+        form.submit()
```

Comment:
This will prevent submitting "remote" forms via XHR, unfortunately, and do a full HTTP request instead. See: https://github.com/rails/rails/blob/5af7d475c8cd7d3dbffca7452f3dcb221896e8e5/actionview/app/assets/javascripts/rails-ujs/features/method.coffee#L21-L23 https://github.com/rails/rails/blob/5af7d475c8cd7d3dbffca7452f3dcb221896e8e5/actionview/app/assets/javascripts/rails-ujs/features/method.coffee#L32

Will need to explore using the `button.focus()` solution you suggested below.